packages/hub-threads-client/src/client.ts

```diff
@@ -66,6 +64,33 @@ Client.prototype.getThread = async function (
   })
 }
 
+// Private listDBs method
+const oldListDBs = Client.prototype.listDBs
+
+/**
+ * Lists all known DBs.
+ */
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+Client.prototype.listDBs = async function (
+  ctx?: Context,
+): Promise<Record<string, GetThreadResponse | undefined>> {
```

Comment:
Proposal, this just "lists" (we could actually use a list) the threads which are dbs. It does _not_ include the key info or addrs. To get this information, a second call to getDBInfo would be required. This seems useful, because often you just want to list DBs, but you only need the extra info if you are going to do something with a given thread. 

Currently in threads-client, this information (addrs and key) are included by default, but since this isn't readily available here, I'm suggesting we drop this... perhaps leaving it backward compatible from threads-client?

Reply:
Nice, makes sense... we can drop from the go-threads API too.